final-project/Figures/api-component.mmd

```diff
@@ -0,0 +1,20 @@
+C4Container
+title Component diagram for APIProxy for OpenTogetherTube
+
+    Person(customer, Customer, "A user of OTT", $tags="v1.0")
+
+    Container_Boundary(c1, "OpenTogetherTube") {
+    Container(browser, "Browser", "Web Browser", "Used by the user to access the application", "v1.0")
+    Container(server, "Server", "Java", "Serves as the backend for the OpenTogetherTube platform", "v1.0")
+    Container(redis, "Redis", "In-memory data structure store", "Stores session information for authenticated users", "v1.0")
+}
```

Comment:
Missing Postgres, which is used for User, and Room persistance, and caching video metadata.